example/caffe/code_samples/caffe_sample.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+import sys
+
+
+# Get the command line arguments
+model_file = ''
+try:
+    model_file = sys.argv[1]
+except IndexError:
+    print('Usage: python caffe_sample.py PATH_TO_MODEL')
+    exit()
+
+# Create solver.prototxt
+with open('solver.prototxt', 'w') as file:
+    # path to the network
+    file.write('net: "{}"\n'.format(model_file))
```

Comment:
@sotanishy rather than using file.write for every parameter, put all the parameters in one json or array and then use it. This code is tough to interpret.